guide/controller-plugin.md

```diff
@@ -195,7 +195,7 @@ In the motion editor different fonts and colors are used to identify the status
 
 %end
 
-### Remote-control Plugin
+### Remote-Control Plugin
```

Comment:
Inconsistency: sometimes we write "remote control" (without dash) and sometimes, it's "remote-control" (with dash). We should be more consistent about this.

Reply:
To be specific: "Remote Control" has no hyphen. "Remote-controlled" (using a passive form of control) needs a hyphen.

Reply:
I propose to fix this in another PR, because it goes quite far (I just checked and there are 130 references to remote control in various states).